README.md

```diff
@@ -599,10 +601,10 @@ Q4: What does this error mean - `"We currently do not support adding policies fo
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 3.61 |
-| <a name="provider_external"></a> [external](#provider\_external) | >= 1 |
-| <a name="provider_local"></a> [local](#provider\_local) | >= 1 |
-| <a name="provider_null"></a> [null](#provider\_null) | >= 2 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 3.64.2 |
```

Comment:
I believe all these changes not related to the addition of `pip_tmp_dir` should not be included in this PR.

@antonbabenko that's an issue I've always faced when running the terraform-docs pre-commit hook in this repository, but never asked about: what are we supposed to do about that?

Reply:
I was following the instructions on another PR. But I also don't know what to do with this one. It happened just when running the hooks.

Reply:
What seems especially wrong to me is replacing the flexible constraints coming from `versions.tf` by hard-coded version numbers. I believe this is caused by the presence of `.terraform.lock.hcl` files.

Probably caused by the fact that we are using too recent (>= 0.14) versions of terraform-docs that include https://github.com/terraform-docs/terraform-docs/pull/509, while the [Github Actions CI is using v0.13.0](https://github.com/terraform-aws-modules/terraform-aws-lambda/blob/v2.26.0/.github/workflows/pre-commit.yml#L94).

Note: terraform-docs v0.15 introduced a `--lockfile` parameters that allows to disable that behavior: https://github.com/terraform-docs/terraform-docs/pull/527

Reply:
Skip this issue, please. I know it was problematic for many for some time. I will fix the docs to make checks green if necessary after this PR is ready.

We are going to fix it in the near future like it is done here - https://github.com/terraform-aws-modules/terraform-aws-rds/pull/361